saleor/core/taxes/avatax/__init__.py

```diff
@@ -18,10 +18,16 @@
     from ....checkout.models import Checkout
     from ....order.models import Order
 
-META_FIELD = "avatax"
-
 logger = logging.getLogger(__name__)
 
+META_FIELD = "avatax"
+CACHE_TIME = 60 * 60
+TAX_CODES_CACHE_TIME = 60 * 60 * 24 * 7
+COMMON_CARRIER_CODE = "FR020100"
```

Comment:
Might want to add a comment explaining what this value is? (would save time when one reads the code)